en-US/advanced/configuration_for_source_bulids.md

```diff
@@ -0,0 +1,236 @@
+---
+name: Detailed configuration for source builds
+sort: 7
+---
+
+Following the instruction of building [from source](/docs/en-US/installation/install_from_source.md), you should now have a local user `git` and a working setup of `go` and [gigs](http://gogs.io/).
+We are gong to setup 'gigs' along with **nginx**. This way, we can take adavantage of nginx. This is useful for servers that already have an nginx running.
```

Comment:
1. typo? `gigs`
2. For my OCD, would recommend to use `Nginx`(capital **N**) in all places. :smile: